readme.md

```diff
@@ -59,6 +59,20 @@ Default: `[]`
 
 Exclude keys from being camel-cased.
 
+##### excludePaths
+
+Type: `Array<string>`<br>
+Default: `[]`
+
+Exclude paths from being camelCased.
```

Comment:
The use case is simple:
```js
{
  a_b: 123,
  a_c: { 
    c_d: 123,
    c_e: {
      e_f: 123,
    },
  },
}
```
becomes
```js
{
  aB: 123,
  aC: { 
    cD: 123,
    c_e: {
      eF: 123,
    },
  },
}
```
with `excludePaths` set to `['a_b.c_e']`.

If you only have `stopPaths`, the you can only make
```js
{
  aB: 123,
  aC: { 
    cD: 123,
    c_e: {
      e_f: 123,
    },
  },
}
```

IMO, having such option is really good in terms of completeness of the lib, as it allows for complete control over every field without additional manipulations(and it costs practically nothing to implement).

Reply:
I struggle to explain what paths are in less than a page of text. On the other side, we have `dot-prop`, which got 7m downloads weekly, works only with paths and doesn't have any descriptions of what paths are, but it seems like everyone understands.

Reply:
Perhaps it would be more clear to specify "object paths" rather than just "paths"? For example:

> Object paths (in string formatted dot-notation) to exclude from being camelCased. The exclusion also applies to all children of the specified paths.